test/test_helper.rb

```diff
@@ -78,6 +78,10 @@ def refute_valid(record, error_keys = nil)
     end
   end
 
+  def freeze_time
```

Comment:
a bit misleading ... would have expected this to only do a `Timecop.freeze` ... but I guess this is ok